test/go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	k8s.io/klog/v2 v2.110.1
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/cluster-api v1.6.5
```

Comment:
how about this one ?
/hold

Reply:
I was expecting two PRs like in other branches. Maybe we should add grouping for cluster-api?

Reply:
true. 
/hold cancel